addons/api/package.json

```diff
@@ -23,59 +25,63 @@
     "doc:toc": "doctoc README.md"
   },
   "dependencies": {
+    "@babel/core": "^7.26.10",
+    "@ember/string": "^4.0.0",
     "dexie": "^3.2.4",
     "ember-auto-import": "^2.10.0",
     "ember-can": "^4.2.0",
     "ember-cli-babel": "^8.2.0",
-    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-mirage": "^3.0.3",
     "ember-data": "~5.3.12",
     "miragejs": "^0.1.48",
     "tracked-built-ins": "^3.1.1"
   },
-  "peerDependencies": {
-    "sinon": "^19.0.2"
-  },
   "devDependencies": {
-    "@babel/core": "^7.26.10",
-    "@babel/eslint-parser": "^7.21.3",
-    "@babel/plugin-proposal-decorators": "^7.21.0",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7",
+    "@ember/optional-features": "^2.1.0",
     "@ember/test-helpers": "^4.0.5",
     "@ember/test-waiters": "^4.1.0",
-    "@embroider/test-setup": "^2.1.1",
+    "@embroider/test-setup": "^4.0.0",
     "@faker-js/faker": "^8.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^9.1.0",
     "doctoc": "^2.2.0",
-    "ember-cli": "^5.12.0",
+    "ember-cli": "~5.12.0",
+    "ember-cli-clean-css": "^3.0.0",
     "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-exam": "^8.0.0",
     "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
+    "ember-page-title": "^8.2.3",
     "ember-qunit": "^8.1.0",
-    "ember-resolver": "^10.0.0",
-    "ember-source": "~4.12.0",
+    "ember-resolver": "^12.0.1",
+    "ember-source": "~5.12.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.7.2",
-    "eslint": "^8.54.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-ember": "^11.11.1",
-    "eslint-plugin-n": "^16.3.1",
-    "eslint-plugin-qunit": "^8.0.1",
+    "ember-template-lint": "^6.0.0",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.2.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.2.1",
```

Comment:
Was this automatically added?

Reply:
Yup changes in package.json are a combination of changes from running `ember-cli-update` and `ember init` for blueprint updates (tho there is due diligence of looking through each change). So `eslint-plugin-prettier` along with `'plugin:prettier/recommended',` is a change I kept as part of blue print changes. Mainly because I didn't realize we did not want prettier to run as part of eslint. I'll revert this change and prettier config.